site/app/controllers/student/SubmissionController.php

```diff
@@ -1020,6 +1020,75 @@ public function ajaxRegrade($gradeable_id): array {
         return $this->core->getOutput()->renderResultMessage($msg, true);
     }
 
+    /**
+     * @return JsonResponse
+     * @param string $gradeable_id
+     */
+    #[Route('/api/{_semester}/{_course}/gradeable/{gradeable_id}/values', methods: ['GET'])]
+    public function ajaxGetGradeableValues(string $gradeable_id): JsonResponse {
+        if ($this->core->getQueries()->getUserById($_GET['user_id']) === null) {
```

Comment:
You're using `$_GET['user_id']` in multiple places, but not always checking that it is not null.  You could consider making a temporary variable instead.